src/mqt/predictor/rl/predictorenv.py

```
@@ -71,22 +72,33 @@ def __init__(
             max_steps: The maximum number of actions per episode. Defaults to None, which means no step limit is enforced.
             tracer_output_path: Path to export the compilation trace JSON. Defaults to None.
+            mdp: The MDP transition policy. ``v2`` is the original MQT Predictor
+                strategy. ``v3`` is the default and is flexible before layout while
+                preserving the established compilation structure afterwards.
+                ``flexible`` provides the greatest freedom among all structurally
+                valid actions.
 
         Raises:
             ValueError: If the reward function is "estimated_success_probability" and no calibration data is available for the device or if the reward function is "estimated_hellinger_distance" and no trained model is available for the device.
         """
         logger.info("Init env: " + reward_function)
```

Ruff (0.16.1)

[warning] Logging statement uses `+`

(G003)